packages/schematics/angular/class/schema.json

```diff
@@ -16,7 +17,7 @@
     "path": {
       "type": "string",
       "format": "path",
-      "description": "The path to create the class.",
+      "description": "The path at which to create the class, relative to the workspace root.",
```

Comment:
Hmmm...."The folder in which to create.."? "The path for the folder in which to create.."? (ditto hmmm on other similar phrases below)